Resources/config/schema/seo-1.0.xsd

```diff
@@ -18,10 +18,9 @@
         <xsd:attribute name="title" type="xsd:string" />
         <xsd:attribute name="description" type="xsd:string" />
         <xsd:attribute name="original-route-pattern" type="original_route_pattern" />
-        <xsd:attribute name="content-key" type="xsd:string" />
```

Comment:
@WouterJ do we need to bump the schema version? or keep this in the deprecated location too, as we have BC to still handle it in the Configuration class?